xds/internal/balancer/clusterresolver/resource_resolver_test.go

```diff
@@ -24,9 +24,10 @@ import (
 	"testing"
 
 	"github.com/google/go-cmp/cmp"
+	"google.golang.org/grpc/internal/testutils"
 	"google.golang.org/grpc/resolver"
 	"google.golang.org/grpc/resolver/manual"
-	"google.golang.org/grpc/xds/internal/testutils"
+	testutilsXDS "google.golang.org/grpc/xds/internal/testutils"
```

Comment:
Could you please rename the import as `xdstestutils` instead. We have done that in other places, and doing the same here wold stay consistent with existing usages. Thanks.

Reply:
Done!

Thanks for the explanation. Makes sense.